zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/notifier/IncidentNotifierTest.java

```diff
@@ -87,7 +87,7 @@ public void setup() {
     final IncidentNotifierConfiguration configuration = new IncidentNotifierConfiguration();
     configuration.setWebhook(ALERT_WEBHOOKURL_URL);
     incidentNotifier =
-        new IncidentNotifier(m2mTokenManager, processCache, configuration, httpClient);
+        new IncidentNotifier(m2mTokenManager, processCache, configuration, httpClient, null);
```

Comment:
🔧 I think it is good that we have DI for the executor. We could use this now, even to validate what happens, when the execution fails, when it is stuck, etc. would it block other execution, etc.?